src/pal/pal_apple.h

```diff
@@ -205,7 +205,7 @@ namespace snmalloc
     // Apple's `mmap` doesn't support user-specified alignment and only
     // guarantees mappings are aligned to the system page size, so we use
     // `mach_vm_map` instead.
-    template<bool committed>
+    template<bool state_using>
```

Comment:
Should this be a tri-state `enum`?  Are there cases where we want to provide read-only access to everything (e.g. the entire pagemap)?

Reply:
There aren't currently cases.  I propose adding if it is required.